cmd/notation/plugin.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func pluginCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "plugin",
+		Use:   "plugin [command]",
```

Comment:
I think this add is not necessary? Cobra would automatically print out [command] when calling `notation plugin -h`

Reply:
Just want to make it clear, but removed now.